src/errors.rs

```diff
@@ -91,6 +91,15 @@ impl MomentoError {
             details: None,
         }
     }
```

Comment:
i know this wasn't part of this PR but why do we have an error for `miss`?

Reply:
That's used when you call `try_into` on a response that wasn't a hit.